java-api/src/main/java/com/sap/cloud/security/token/TokenClaims.java

```diff
@@ -46,7 +54,7 @@ private XSUAA() {
 
 		public static final String ORIGIN = "origin";
 		public static final String GRANT_TYPE = "grant_type"; // OAuth grant type used for token creation
-		public static final String ZONE_ID = "zid"; // tenant GUID -> SAP_GLOBAL_ZONE_ID
+		public static final String ZONE_ID = "zid"; // tenant GUID -> SAP_GLOBAL_APP_TID
```

Comment:
I don't understand the comment. Does it mean I should always use SAP_GLOBAL_APP_TID instead of this?

Reply:
it means the value is == to SAP_GLOBAL_APP_TID, but i'll rephrase the comment